connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -306,6 +308,19 @@ public static synchronized void sdkInitialize(Context context) {
                 sRedirectUri);
     }
 
+    public static String getMccMncLoginHint() {
```

Comment:
Consider splitting this method into getMccMnc() and getMccMncLoginHint(String mccMnc), so we can use both.